README.md

```diff
@@ -219,7 +219,7 @@ Variables in vars.yml
    -  ** `type: files`**
       - `facility`: facility; default to `*`
       - `severity`: severity; default to `*`
-      - `exclude`: exclude; default to none.
+      - `exclude`: exclude; default to none. exclude list must start with ';' and is delimited by ';'. E.g., ';cron.none;mail.none'
```

Comment:
Why to introduce this format? We are using YAML, so we don't have to use strings with embedded delimiters and parse them into lists. We can use lists directly:
```yaml
exclude:
  - cron.none
  - mail.none
```

Reply:
Ok, let me update the patch according to your proposal. Thanks!